website/src/docs/hotchocolate/filtering/list-filter.md

```diff
@@ -0,0 +1,125 @@
+### List Filter
```

Comment:
Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.